rpcd/playbooks/ansible.cfg

```diff
@@ -1,5 +1,8 @@
 [defaults]
 gathering = smart
+fact_caching = jsonfile
```

Comment:
We also need the facts when doing --limit as each ceph MON needs access to the other MONs to fully populate /etc/ceph/ceph.conf.